### PR TITLE
Changes how the Slider works in vertical orientation

### LINKED ISSUE #5054: fix: vertical mode in slider in the @microsoft/fast-components package
# 🐛 Bug Report

For LTR languages I would expect that slider would place the minimum value at the bottom of the range and the maximum value at the top. This appears to be inverted. I do not know what I would expect for RTL, but it also appears that those values are inverted when switching.

## 💻 Repro or Code Sample

Code:
Screen Shot 2021-08-11 at 11 26 28 AM

LTR vertical result:
Screen Shot 2021-08-11 at 11 26 22 AM

RTL vertical result:
Screen Shot 2021-08-11 at 11 26 43 AM

## 🤔 Expected Behavior

I would expect the current result for LTR to be the RTL and vice versa.

### PR DESCRIPTION
**Pull request by procload**
# Pull Request

## 📖 Description
Closes #5054 by changing the direction of the Slider thumb to instead increase value when sliding upward. This matches the native slider element when in vertical orientation.

### 🎫 Issues
#5054 

## 👩‍💻 Reviewer Notes

The slider should remain the same except for how the value is increased or decreased when it is in a vertical orientation.

## 📑 Test Plan

No tests were affected by this change

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by chrisdholt — CHANGES_REQUESTED** [review]
I pulled this down and I noticed that the keyboarding behavior also needs to be updated - FWIW, it seems like this is broken today. Up should increment, but the actual value (and visuals) are going down. Similarly, down should be decrementing but value and visual is going up. Would be great to get a holistic fix in that addresses the keyboarding behavior as well. Let's be sure to add tests if we can to validate that keyboarding increments and decrements as expected.
image

**Comment by procload** [comment]
> I pulled this down and I noticed that the keyboarding behavior also needs to be updated - FWIW, it seems like this is broken today. Up should increment, but the actual value (and visuals) are going down. Similarly, down should be decrementing but value and visual is going up. Would be great to get a holistic fix in that addresses the keyboarding behavior as well. Let's be sure to add tests if we can to validate that keyboarding increments and decrements as expected. image

I saw the tests to check `increment()` and `decrement()` in the file, but didn't see anything related to left/right/up/down or any other keyboard commands. It looks like the keypresses are calling these methods. Should I go forward with adding tests for these, and if so, what should I be testing for outside of if the methods fire?

**Comment by chrisdholt** [comment]
> I saw the tests to check `increment()` and `decrement()` in the file, but didn't see anything related to left/right/up/down or any other keyboard commands. It looks like the keypresses are calling these methods. Should I go forward with adding tests for these, and if so, what should I be testing for outside of if the methods fire?

Check out the tests in Menu here: https://github.com/microsoft/fast/blob/d0ef0d6b1540ed90730187b45bdaf47aa9dcb183/packages/web-components/fast-foundation/src/menu/menu.pw.spec.ts#L327

I'd expect tests to cover the outcomes for behavior for both modes:
- it should increment the value when up arrow is pressed
- it should increment the value when right arrow is pressed
- it should decrement the value when down arrow is pressed
- it should decrement the value when left arrow is pressed

If we had the above currently we'd know that in vertical it's not working, so the goal is to ensure we fix and have that coverage :)

File: change/@microsoft-fast-foundation-19bde5a4-4383-4eab-9191-79bdf549c6f3.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Changes how the Slider works in vertical orientation by adjusting the slider to increase value when the thumb is moved upwards",
```
Comment (chrisdholt) [inline change/@microsoft-fast-foundation-19bde5a4-4383-4eab-9191-79bdf549c6f3.json:3]:
```suggestion
"comment": "Fixes Slider vertical orientation so adjusting the slider increases the value when the thumb is moved upwards and corrects keyboard handling",
```

**Review by chrisdholt — APPROVED** [review]

**Review by eljefe223 — APPROVED** [review]

**Pull request merged**